guides/source/configuring.md

```diff
@@ -865,6 +865,8 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.use_authenticated_message_encryption` specifies whether to use AES-256-GCM authenticated encryption as the default cipher for encrypting messages instead of AES-256-CBC.
 
+* `config.active_support.cache_format_version` specifies which version of the cache serializer to use. Possible values are `6.1` and `7.0`. Defaults to `6.1`.
```

Comment:
Is the default value correct?

Reply:
Yes, by default it keeps generating the old format. You use to use `load_defaults 7.0` for it to write the new format.

This is so that you can first deploy Rails 7.0 with `load_defaults 6.1`. Otherwise if you directly deploy your Rails upgrade, and it start writing the new format, during the rollout some old processes still on Rails 6.1 won't be able to read the cache written by the new processes.

Reply:
So, the default depends really on what you have in `load_defaults`, right? That is my point.

Reply:
With `zeitwerk` mode it was the same. It is the default in Rails 6 apps _that loaded 6.0 defaults_, if you have a Rails 6 app loading 5.2 defaults, you get `classic`.